pages/components/breadcrumbs.html

```diff
@@ -0,0 +1,22 @@
+<html lang="en">
+    <%- include('head.html'); -%>
+    <body>
+        <%- include('nav.html'); -%>
+
+        <main>
+            <h1>&lt;fabric-breadcrumbs&gt;</h1>
+
+            <h2>Examples</h2>
+
+            <fabric-breadcrumbs class="mt-10">
```

Comment:
I noticed that any styles applied inline or through classes don't actually appear on the custom element.

It appears to be because the custom element parent not acting like a div. Styles appeared when I changed the `<fabric-breadcrumbs>` to`<div>` through the DOM.

One solution here can be to append the styles from the custom element directly onto the nav we inject, just like we do with the children. Thoughts?

```this.innerHTML = `<nav class="flex space-x-8 ${this.classList}">${children}</nav>`;```

Reply:
Other than that, very nice and tiny implementation 😄

Reply:
Oh good catch! This is probably because I forgot to set the :host to be a block level element.

Reply:
Ah, yeah! I see it's done differently with the box component, but would a one-liner like so do? `this.style.display = `block`;` Or is using the pseudo-class a preferred method?

Reply:
I'm a bit unsure on this. I suspect that the pseudoclass is the preferred method though my only thought as to why that might be could have something to do with overriding user set syles. I'm unsure what happens if we set the style directly internally and at the same time the user sets the display on the element externally.

Reply:
:host won't work without Shadow DOM it seems so I think we'll have to check if display is set and if not, set it to block

Reply:
Or perhaps we should be leaving this up to the user entirely. @Skadefryd @pearofducks what do you guys think? Should a custom element have an initial display set? `block` vs `inline` etc? Or should it just be up to the user to set. (no choice essentially means inline)

Reply:
I would think most people expect `block` for these components. `inline` might really throw people - f.ex if they try to set a top margin and that doesn't work.

Reply:
Agree. That makes good sense.